### DISCUSSION TITLE
2022 年の振り返りレポートを入手したい

### DISCUSSION THREAD

**Discussion by punkshiraishi** [increments/qiita-discussions#402] [ヘルプ]
振り返りレポートというものの存在を知ったのが 12/28 日の配信後でした。それまではメール通知を OFF にしていたので、私には届かなかったようです。

今からでも入手する方法はありますでしょうか。

**Comment by yoshi389111** [comment]
通りすがりの者です。

単なる参考情報ですが。
去年も同じような要望があって、あとからフォームを設置して対応していたようです。
（要望が多ければ）今年もやってくれるんじゃないかな？

* [2021年の振り返りレポートをWebで見れますか？](https://github.com/increments/qiita-discussions/discussions/100)
* [Qiita 2021年振り返りレポートを受け取れなかった方へ](https://github.com/increments/qiita-discussions/discussions/101)

無事振り返りレポートを入手できるといいですね。

**Reply by punkshiraishi** [reply]
ありがとうございます。今年もやってくれることを期待したいです...！

**Comment by tanaka-asami415** [comment]
@punkshiraishi 
お問合せありがとうございます！

今回は事前に告知を行っていたこともあり、追加配信は予定していなかったのですが
とても楽しみにしてくださっているようなので、今回が最後…ということで、
追加配信をさせていただきます！✨

準備ができましたら、アナウンス用のDiscussionでお知らせさせていただきます！🙇

**Reply by tanaka-asami415** [reply]
@punkshiraishi 
お待たせいたしました！こちらからお申込みいただけると幸いです🙇
https://github.com/increments/qiita-discussions/discussions/412

**Reply by punkshiraishi** [reply]
ありがとうございます！申し込ませていただきます。
